optimize_workspace.sh

```diff
@@ -27,10 +32,9 @@ TMPARTIFACTS=$(mktemp -p "$(pwd)" -d artifacts.XXXXXX)
   cd "$TMPARTIFACTS"
 
   for WASM in ../target/wasm32-unknown-unknown/release/*.wasm; do
-    echo "Optimizing $WASM ..."
-    BASE=$(basename "$WASM")
-    wasm-opt -Os -o "$BASE" "$WASM"
-    chmod -x "$BASE"
+    NAME=$(basename "$WASM" .wasm)${SUFFIX}.wasm
+    echo "Optimizing $NAME ..."
+    wasm-opt -Os "$WASM" -o "$NAME"
```

Comment:
Now the resulting `$NAME`s are executable, aren't they? At least we needed `chmod -x` before.

Reply:
okay, cool, seems to work on Mac as well. Maybe it was an old bug that disappeared.